README.md

```diff
@@ -210,7 +208,7 @@ test runner.
 You can run all tests with:
 
 ```bash
-$ npm run test
+$ yarn run test
```

Comment:
I pushed a fixup(51abd82) and now we use `yarn run` on all scripts described in `package.json`.